src/screens/points/contexts/PointsProfileContext.tsx

```diff
@@ -167,7 +167,8 @@ export const PointsProfileProvider = ({
     const challenge = challengeResponse?.pointsOnboardChallenge;
     if (challenge) {
       const provider = await getProviderForNetwork(Network.mainnet);
-      const wallet = await loadWallet(accountAddress, true, provider);
+      const wallet =
+        (await loadWallet(accountAddress, true, provider)) ?? undefined;
```

Comment:
```suggestion
if (!wallet) {
        logger.error(new RainbowError('Error loading wallet'), {
          referralCode,
          challenge,
          accountAddress,
        });
        Alert.alert(i18n.t(i18n.l.points.console.generic_alert));
        return;
}
```

I'd prefer to fail instead of proceeding if we're unable to load the wallet for whatever reason